client/state/index.js

```diff
@@ -71,6 +72,7 @@ export const reducer = combineReducers( {
 	componentsUsageStats,
 	countryStates,
 	currentUser,
+	directly,
```

Comment:
Sure, nesting this makes sense. Directly is similar to Olark and Happychat in purpose (live chat support) and I see those are both in `ui` — does `ui/directly` sound reasonable?